platform-sdk/consensus-otter-tests/src/testFixtures/java/org/hiero/otter/fixtures/app/state/OtterStateInitializer.java

```diff
@@ -37,7 +37,7 @@ public static void initOtterAppState(
             final Schema<SemanticVersion> schema = service.genesisSchema(version);
             for (final StateDefinition<?, ?> stateDefinition : schema.statesToCreate()) {
                 // the metadata associates the state definition with the service
-                final StateMetadata<?, ?> stateMetadata = new StateMetadata<>(service.name(), schema, stateDefinition);
+                final StateMetadata<?, ?> stateMetadata = new StateMetadata<>(service.name(), stateDefinition);
```

Comment:
This is great! If we no longer need the `Schema` class for `StateMetadata`, I think we can eliminate `Schema` and `MigrationContext` entirely in Otter. They never fitted anyway. (cc: @poulok )